etc/databases/world/updates/updates.sql

```diff
@@ -30417,5 +30417,29 @@ begin not atomic
 
         insert into applied_updates values ('050320231');
     end if;
+
+    -- 06/03/2023 2
+    if (select count(*) from applied_updates where id = '060320232') = 0 then
+        -- Update name requirements and quest text for quest 951 to what it was like during alpha.
+        update quest_template set Title = 'Blackfathom Relics', ReqItemId2 = 5330, ReqItemId3 = 5331, ReqItemCount1 = 2, ReqItemCount2 = 2, ReqItemCount3 = 2, Objectives = 'Bring 2 Blackfathom Relics, 2 Mathystra Relics and 2 Elven Cup Relics to Onu at the Grove of the Ancients.', OfferRewardText = 'Thank you. These relics are from a time when Mathystra shone. That once great bastion of the elves has faded, but shards of its magic remain. Let us hope we can unlock the secrets of that place before our enemies...$B$BBe vigilant, $N. And prepare yourself to return to the deeps within the Ruins of Mathystra. Fate will, one day, take you there.' where entry = 951;
+
+        -- Reactivate Blackfathom Relic (5331).
+        update item_template set name = '[PH] Blackfathom Relic', display_id = 8290, stackable = 2, max_count = 2, bonding = 2, flags = 0, buy_price = 0, sell_price = 0 where entry = 5331;
+        -- Change Mathystra Relic (5273) and Elven Cup Relic (5330) to their alpha form.
+        -- Since the original "green square" placeholder icon for 5273 is not present in the client we'll use INV_Misc_Questionmark (display_id 1240).
+        update item_template set name = '[PH] Mathystra Relic', display_id = 1240, stackable = 2, max_count = 2 where entry = 5273;
+        update item_template set name = '[PH] Elven Cup Relic', display_id = 926, stackable = 2, max_count = 2 where entry = 5330;
+
+        -- Restore "Mathystra Relic" gameobject loot tables to the original values.
+        update gameobject_template set data1 = 2770 where entry = 13360;
+        update gameobject_template set data1 = 2769 where entry = 12654;
+        update gameobject_template set data1 = 2767 where entry = 13872;
```

Comment:
I think this is swapped; `13872` is the cup gameobject that should contain `[PH] Elven Cup Relic` (`5330`), but `gameobject_loot_template` `2767` is set to drop `5273` which is is `[PH] Mathystra Relic`. Gameobject `13360` is the Night Elf statue that's already dropping `[PH] Blackfathom Relic` (`gameobject_loot_template` `2767`).

Also, they should probably be renamed to the item they drop, they are currently all named `Mathystra Relic`.

Reply:
You did see that I added two more loot tables? Only one (2767) still existed because the quest was reworked to only require Mathystra relics and the data is from vanilla.

Reply:
> You did see that I added two more loot tables? Only one (2767) still existed because the quest was reworked to only require Mathystra relics and the data is from vanilla.

Yup! But gameobject `13360` is already assigned to the correct loot table (`2767`) I think.

Reply:
Nevermind I got confused, it's correct. :+1: